javascript/javascript3/week3/promises.js

```diff
@@ -0,0 +1,44 @@
+let container = document.getElementById('container');
+
+
+function fetchDataUser(name) {
+    fetch(`https://api.github.com/search/repositories?q=user:${name}`)
```

Comment:
cleaner with async / await, but great that you avoided using "new Promise" here!